man/docker-build.1.md

```diff
@@ -92,8 +92,14 @@ set as the **URL**, the repository is cloned locally and then sent as the contex
 **-m**, **--memory**=*MEMORY*
   Memory limit
 
-**--memory-swap**=*MEMORY-SWAP*
-  Total memory (memory + swap), '-1' to disable swap.
+**--memory-swap**=*LIMIT*
+   A limit value equal to memory plus swap. Must be used with the  **-m**
+(**--memory**) flag. This value should always larger than **-m**(**--memory**)
+value.
+
+   The format of `LIMIT` is <`number`>[<`unit`>]. Unit can be `b` (bytes),
```

Comment:
can you put the back-tics around the whole "format" here?

    The format of `LIMIT` is `<number>[<unit>]`. Unit can be `b` (bytes),